config/initializers/rails_admin.rb

```diff
@@ -433,7 +439,14 @@
       field :feeds
     end
     edit do
-      field :slug
+      field :collection do
+        visible do
+          true unless bindings[:object].persisted?
+        end
+        associated_collection_scope do
+          proc { |_scope| Collection.published.includes(:page_landing).where(pages: { collection_id: nil }) }
```

Comment:
I think it's a bit tidier to do `proc { |scope| scope.published.includes(:page_landing).where(pages: { collection_id: nil }) }`